src/utils/buildAggregateOffer.ts

```diff
@@ -0,0 +1,11 @@
+import { AggregateOffer } from '../types';
+
+export default (offer: AggregateOffer) => `
```

Comment:
Could you change this to be a named export. 

```
export const  buildAggregateOffer(
```

Reply:
Ofcourse! I have updated it 👍🏼